src/cmd/util_test.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Operation string
+
+const (
+	Assign   Operation = "assign"
+	Create   Operation = "create"
+	Delete   Operation = "delete"
+	Example  Operation = "example"
+	Get      Operation = "get"
+	List     Operation = "list"
+	Update   Operation = "update"
+	Unassign Operation = "unassign"
+)
+
+// execute any OpsLevel CLI command
+func execCmd(cmd Operation, resource string, inputs ...string) ([]byte, error) {
```

Comment:
Could we do this? 

Rename `execCmd` to `ExecCmd` and keep it in the `cmd` package (since it needs access to `rootCmd`), and then remove everything else added in this PR a new `cmd_test` package?

This would give us better separation between CLI code and tests. However the CLI is not really a "library" we are providing so I don't hold this opinion strongly.